template/bacardi_cpp.njk

```diff
@@ -17,12 +17,17 @@
 #ifndef EXAMPLES_BACARDI_H_
 #define EXAMPLES_BACARDI_H_
 
-#include "examples/bacardi.h"
-#include "examples/calculator_bridge.h"
+#include "core/bacardi.h"
 
+{% for name in names %}
+#include "examples/{{name.name | lower}}_bridge.h"
+{% endfor %}
+
+{% for name in names %}
```

Comment:
i think present |names| have one item. so it was not made some problem in build time.
if names were had one more items. it will be make some problem because it has same name functions(parameters is same)

so i think iterator will be move to inside of Init function.